tests/ops/op_math/test_prod.py

```diff
@@ -733,6 +633,11 @@ def test_depth_property(self):
         )
         assert prod_op.arithmetic_depth == 3
 
+        op_constructed = Prod(
+            qml.RZ(1.32, wires=0), qml.Identity(wires=0), qml.RX(1.9, wires=1), qml.PauliX(0)
+        )
+        assert op_constructed.arithmetic_depth == 1
```

Comment:
unfortunate discrepancy. same as in test_sum.py :/

Reply:
Ahh this was due to the lack of simplify ?

Reply:
well, you could simplify the one before and it would also have a depth of 1 :) just no one did it in this test